package/MDAnalysis/lib/formats/xdrlib.pyx

```diff
@@ -490,7 +490,7 @@ cdef class TRRFile(_XDRFile):
             The arguments do not match with previous saved frames.
         """
         if self.mode != 'w' :
-            raise RuntimeError('File opened in mode: {}. Writing only allow '
+            raise IOError('File opened in mode: {}. Writing only allow '
```

Comment:
I changed the RuntimeError to IOErrors here as well so this will conflict with #691